packages/vscode-extension/src/resources/Strings.json

```diff
@@ -39,6 +39,7 @@
       "validateManifestTitle": "Validate App Manifest File",
       "validateManifestTitleNew": "Validate manifest file",
       "validateManifestDescription": "Validate the Teams app manifest file",
+      "addApiConnectorTitle": "Connect to an API",
```

Comment:
It seems that this variable is not used? The localized string is stored in `package.nls.json`

Reply:
removed